src/main/java/org/zalando/nakadi/service/AuthorizationValidator.java

```diff
@@ -155,7 +155,9 @@ public void authorizeStreamRead(final EventType eventType) throws AccessDeniedEx
         final Resource resource = new EventTypeResource(eventType.getName(), eventType.getAuthorization());
         try {
             if (!authorizationService.isAuthorized(AuthorizationService.Operation.READ, resource)) {
-                throw new AccessDeniedException(AuthorizationService.Operation.READ, resource);
+                if (!adminService.hasAllDataAccess(AuthorizationService.Operation.READ)) {
```

Comment:
Maybe it is normal in Java, just curious, Why two IF instead of AND?

Reply:
The `&&` does short-circuit evaluation so actually it would be equivalent

Reply:
It would be nicer with a `&&`, you are right

Reply:
7927b54